semcore/ellipsis/src/index.d.ts

```diff
@@ -0,0 +1,39 @@
+import { CProps } from '@semcore/core';
+import { RefObject } from 'react';
+
+export interface IEllipsisProps {
+  /**
+   * Row count in multiline Ellipsis
+   * @default 1
+   */
+  maxline?: number;
+  /**
+   * Trimming type
+   * @default right
+   */
+  trim?: 'end' | 'middle';
+  /**
+   * Shows tooltip
+   * @default visible
+   */
+  tooltip?: string;
+  /**
+   * ref to the item that will be observed by ResizeObserver
+   */
+  // eslint-disable-next-line ssr-friendly/no-dom-globals-in-module-scope
+  containerRef?: RefObject<HTMLElement | null>;
+  /**
+   * Subscribes elements to ResizeObserver
+   */
+  resizeObserver?: { width: number };
```

Comment:
why?

Reply:
Because the description describes something that **does** subscribing.  But as I see in typing it's totally not about subscribing but about plain object with singular number field.